theme/color.go

```diff
@@ -0,0 +1,55 @@
+package theme
+
+import (
+	"image/color"
+)
+
+// Brighten takes a color and brightens / dims it by the given
+// amount.
+// A value of 1.0 == the same
+// 1.1 = 10% brighter
+// 0.9 = 10% dimmer, etc
+func Brighten(c color.Color, amount float64) color.Color {
```

Comment:
I don't think this should be a public API that we have to support ongoing.
If it needs to be expoted then how about somewhere in `internal` packages?